pvlib/modelchain.py

```diff
@@ -803,7 +803,7 @@ def prepare_inputs(self, times=None, weather=None):
             self.weather['temp_air'] = 20
         return self
 
-    def run_model(self, times=None, weather=None):
+    def run_model(self, times=None, weather=None, pv_ref_device=False):
```

Comment:
I'm opposed to this new keyword argument.  See comments about `weather` below.

Reply:
Understood. I'm looking into the alternatives.

Reply:
I am open to suggestions on what to call `F`, as well as the symbol. Perhaps something more expressive (but wordy!) like "short-circuit current ratio". Likewise for `H`, but note that the formulation of this model in terms of `H` is more of a numerical convenience (a re-scaling that also obviates the need to explicitly specify the reference temperature), whereas `F` is fundamentally linked to an "observable" short-circuit current rather than a "non-observable" photocurrent.

Reply:
Yes, something more wordy would be good.  Ideas:
effective_current_ratio, i_sc_ratio, i_sc_norm
temp_cell_abs_norm, temp_cell_norm

Reply:
`effective_irradiance_from_isc` perhaps.  `temp_cell_normalized` is ok with me.